system/settings/settings.xml

```diff
@@ -49,6 +49,9 @@
         <setting id="videoplayer.usedisplayasclock" type="boolean" label="13510" help="36166">
           <level>1</level>
           <default>false</default>
+          <dependencies>
+            <dependency type="enable" setting="audiooutput.passthrough" operator="is">false</dependency>
```

Comment:
paplayer or other clients may want to use pt, while videoplayer does not. pt is a 2nd class citizen and must not have impact on videoplayer features.